examples/demo_2d_swe.py

```diff
@@ -0,0 +1,63 @@
+
+from __future__ import division # Get proper divison
+
+import math
+import random
+
+import numpy as np 
+
+
+from firedrake import *
+parameters["reorder_meshes"] = False
```

Comment:
I think this has always been in my scripts from the first test script I was given. I don't know what it does so I'll delete if unnecessary.